projects/packages/image-cdn/src/class-image-cdn.php

```diff
@@ -333,6 +333,11 @@ public static function parse_dimensions_from_filename( $src ) {
 	 * @return string
 	 */
 	public static function filter_the_content( $content ) {
+		// Early return if content is null, empty, or not a string.
+		if ( null === $content || ! is_string( $content ) || '' === $content ) {
```

Comment:
The null check is redundant, no?

```suggestion
		if ( ! is_string( $content ) || '' === $content ) {
```

Reply:
Very true, thanks!